modules/gdscript/gdscript_parser.h

```diff
@@ -1329,8 +1331,10 @@ class GDScriptParser {
 		PREC_ASSIGNMENT,
 		PREC_CAST,
 		PREC_TERNARY,
+		PREC_LOGIC_XOR,
 		PREC_LOGIC_OR,
 		PREC_LOGIC_AND,
+		PREC_LOGIC_NAND,
```

Comment:
I think the precedence of logical operators should be the same as that of bitwise ones.

https://github.com/godotengine/godot/blob/e0e93ce094d19d132f34aa5c92e892524a8125c9/modules/gdscript/gdscript_parser.h#L1337-L1339

For example, in [PHP](https://www.php.net/manual/en/language.operators.precedence.php) precedence is the same: `or`, `xor`, `and`; `|`, `^`, `&`.